document-loaders/langchain4j-document-loader-azure-storage-blob/pom.xml

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>dev.langchain4j</groupId>
+        <artifactId>langchain4j-parent</artifactId>
+        <version>0.26.0-SNAPSHOT</version>
+        <relativePath>../../langchain4j-parent/pom.xml</relativePath>
+    </parent>
+
+    <artifactId>langchain4j-document-loader-azure-storage-blob</artifactId>
```

Comment:
Sorry for the nitpick, but why not `langchain4j-document-loader-azure-blob-storage`? 😄 as in "LangChain4j Azure Blob Storage document loader" and `AzureBlobStorageDocumentLoader`

Reply:
Ah, that's a naming problem with Azure Blob Storage.
The library's name (as in Maven) is `azure-storage-blob` which makes sense, as there are some other storage mechanisms (like Table Storage and File Storage). And that's also what is being used in URLs like https://learn.microsoft.com/en-us/azure/storage/blobs/
Then, when you say the name in English, people will call it "Azure Blob Storage".

So in the Azure SDK you'll have `com.azure.storage.blob.implementation.AzureBlobStorageImpl`, basically with the package name using "storage-blob" and the Class name using "BlobStorage".

This is what I'm trying to follow here.